### ISSUE TITLE
Incorrect count for Datastore and Network integrations is displayed on Browse tab under Integrations.

### ISSUE THREAD

**Issue by vipulsrivastava-qasource** [elastic/integrations#1350]
**Kibana version:** 7.14 BC-4 Kibana cloud environment

**Host OS and Browser version:** All, All

**Preconditions:**

1. 7.14.0 BC-4 Kibana cloud environment should be available.

```
Build Details:
Build: 42656
Commit: 82a4f6a7fa23946667599787cdb1b2d82c1d0dfb
```

**Steps to reproduce:**

1. Login to Kibana.
2. Navigate to Integrations-> Browse tab.
3. Click on Datastore integration under Browse by category.
4. Observe that incorrect count is displayed for datastore integration.
5. Now, again click on Network integration under Browse by category
6. Observe that incorrect count is displayed for Network integration.

**Actual Result**
Incorrect count for Datastore and Network integrations is displayed on Browse tab under Integrations.

**Expected Result**
Correct count for Datastore and Network integrations is displayed on Browse tab under Integrations.

**What's working**

- This issue is not occurring for other integration under Browse by category.

**What's not working**
N/A

**Screen Recording**

- **Datastore Integration**
![Datastore](https://user-images.githubusercontent.com/69579402/126942122-d6592556-e6e4-4cfb-99fa-73c3a11f536f.png)

- **Network Integration**
![Network](https://user-images.githubusercontent.com/69579402/126942135-21bf7a04-c929-402f-91da-baf4be840af2.png)

**Comment by vipulsrivastava-qasource** [comment]
@manishgupta-qasource : Please review the Bug.

**Comment by manishgupta-qasource** [comment]
Reviewed & Assigned to @jen-huang

**Comment by EricDavisX** [comment]
I'm not sure if this will be Kibana Fleet Side or on the individual Integrations or maybe both.  @jen-huang @masci would you have a guess?  I think we have at least 1 other issue about the counts, found first with the Security selection which I think makes this a dupe if we can find that one.  

I don't consider it a blocker, but I do submit it is kind of embarrassing as Integrations is now so prominent, folks will notice - so if we had an easy fix on the Kibana side that was safe I would be interested in any near term (patch fix even) fix.

**Comment by jen-huang** [comment]
Kibana uses the counts as returned from the package registry so this is indeed an integrations team bug. @kaiyan-sheng I recall you were looking into this?

**Comment by kaiyan-sheng** [comment]
Yes I will push a PR shortly for this and also the security category tag. Here is the issue I have to track this work: https://github.com/elastic/integrations/issues/1342

**Comment by vipulsrivastava-qasource** [comment]
Hi @kaiyan-sheng

We have regressed this issue on below build and found that issue is **Not Fixed** . Could you please share the BC build details for which fix will be available

**Build Details:**

Version: 7.15.0-BC1
Build number: 43636
Commit: d791226d9385122f33f4a5ca38fa5369012fbec3
Artifact: https://staging.elastic.co/7.15.0-d9929120/summary-7.15.0.html

**Moreover, this issue is also occurring for 'Custom' Integration.**

**Screenshot**
![Custom-Integration](https://user-images.githubusercontent.com/69579402/130595062-4157ad8d-880b-4ba3-b0f5-2199c9de6d7e.png)

Thanks!!.

**Comment by kaiyan-sheng** [comment]
@vipulsrivastava-qasource Sorry I'm trying to get the QA deployment to work with 7.15 snapshot. But at the mean time, could you test again with 0.10.4 version please? I just promoted `aws` package 0.10.4 to production yesterday, which should include this change.  
For the `Custom` integration, I'm don't think that's related to the AWS package. This is something different. @jen-huang Do you know who should take a look at this?

**Comment by jen-huang** [comment]
@kaiyan-sheng Didn't we discuss that this might be an issue with the package registry code not counting up packages+integrations categories correctly? Should we move it to that repo instead?

**Comment by kaiyan-sheng** [comment]
@jen-huang Hmmmm for the AWS package, we discussed the problem is with defining categories at the package level or data stream level. For example, we used to have `network` category at the package level for AWS so it's inherited by all data streams underneath. Instead, I remove the `network` category from the AWS package and added only to some individual data streams. 

Maybe for the `Custom` category, this is a different situation?

**Comment by jen-huang** [comment]
@kaiyan-sheng No worries! I did a bit of investigation and the discrepancy comes up when we have these query params: `/categories?experimental=true&include_policy_templates=true`. We're likely double counting policy templates when packages only have 1 policy template.

I will transfer this to package-registry repo.
